kim_convergence/run_length_control/_trajectory.py

```
@@ -0,0 +1,89 @@
+            "int) -> 1darray,\nwhere nstep is the number of steps and the "
+            "function should return a time-series data with the requested "
+            "length equals to the number of steps."
+        )
+
+
+def _get_trajectory(
+    get_trajectory: Callable,
+    run_length: int,
+    ndim: int,
+    number_of_variables: int = 1,
+    get_trajectory_args: dict = {},
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)